sql/moz-fx-data-shared-prod/stripe_external/itemized_tax_transactions_v1/schema.yaml

```diff
@@ -100,7 +100,7 @@ fields:
   type: STRING
   mode: NULLABLE
 - name: filing_exchange_rate
-  type: NUMERIC
+  type: BIGNUMERIC
```

Comment:
It looks like the [bqetl_artifact_deployment](https://workflow.telemetry.mozilla.org/dags/bqetl_artifact_deployment/grid) DAG's `publish_new_tables` task won't be able to update the `stripe_external.itemized_tax_transactions_v1` table's schema in this way using `bqetl query schema deploy`, so I plan to manually update the schema using the following DDL query:
```sql
ALTER TABLE `moz-fx-data-shared-prod.stripe_external.itemized_tax_transactions_v1`
ALTER COLUMN filing_exchange_rate SET DATA TYPE BIGNUMERIC
```